ddtrace/settings/asm.py

```diff
@@ -269,6 +269,12 @@ def _eval_asm_can_be_enabled(self):
     def _api_security_feature_active(self) -> bool:
         return self._asm_libddwaf_available and self._asm_enabled and self._api_security_enabled
 
+    @property
+    def _apm_opt_out(self) -> bool:
+        return (
+            self._asm_enabled or self._iast_enabled or tracer_config._sca_enabled is True
```

Comment:
Do we need this? Can we not just do:
```suggestion
            self._asm_enabled or self._iast_enabled or tracer_config._sca_enabled
```

Reply:
tracer_config._sca_enabled can be None. 

If the _sca_enabled is None and _asm_enabled=True and _iast_enabled=True then `_asm_enabled or _iast_enabled or _sca_enabled == None`. This is wrong. We always want `_apm_opt_out` to return a boolean. Enusuring `tracer_config._sca_enabled` is always a boolean is beyond the scope of this PR.